melodic/distribution.yaml

```diff
@@ -2130,6 +2130,22 @@ repositories:
       url: https://github.com/stack-of-tasks/dynamic-graph.git
       version: devel
     status: maintained
+  dynamic-graph-python:
+    doc:
+      type: git
+      url: https://github.com/stack-of-tasks/dynamic-graph-python.git
+      version: devel
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
```

Comment:
whoops, missed this; I see it's fixed in https://github.com/ros/rosdistro/pull/25965/files